### PR TITLE
update telemetry operator resource

### PR DESCRIPTION
**Pull request by dennis-ge**
**Description**

Changes proposed in this pull request:

- updates the telemetry resource to new LogPipeline CRD
- small updates in the fast integration `telemetry-test`
- needs to be merged after https://github.com/kyma-project/kyma/pull/12910

**Related issue(s)**

### CODE REVIEW

File: resources/telemetry/charts/operator/templates/deployment.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ spec:
           args:
             - --leader-elect
             - --cm-name={{ .Release.Name }}-{{ .Values.logProcessor }}-sections
+            - --parser-cm-name={{ .Release.Name }}-{{ .Values.logProcessor }}-parsers
```
Comment (chrkl) [inline resources/telemetry/charts/operator/templates/deployment.yaml:33]: We should align the parsers volume mount in the fluent-bit chart values with this one. Also double-check the other ConfigMaps.

**Review by chrkl — CHANGES_REQUESTED** [review]

**Review by chrkl — APPROVED** [review]

**Review by lindnerby — APPROVED** [review]

**Pull request merged**